features/step_definitions/journey_test_steps.rb

```diff
@@ -1068,8 +1047,14 @@ def services_listed(service_ids)
   if page_name == 'Digital Marketplace'
     page.visit("#{dm_frontend_domain}")
     page.should have_content("#{page_name}")
+    page.should have_link('Find an individual specialist')
```

Comment:
Are we expecting that DOS will be live from now on whenever the tests are run?  I think that should be fine, although something (the tests?) has been swapping it back to `standstill` recently on preview, in which case this test will fail.